alphagov/paas-elasticache-broker
# [#154332167] Enable automatic daily snapshots

# What

We want to enable the automated daily snapshot feature of elasticache redis.

This features is only available for t2.* node types when "cluster mode" is enabled, so we have decided that we are only supporting "cluster mode" plans going forward so we can safely use this feature for all our plans.

# How to review

* Code review
* Check integration tests
* Try it out via paas-cf: https://github.com/alphagov/paas-cf/pull/1199

# Who can review

Not @chrisfarms nor @henrytk
